### PR TITLE
Add links to arXiv paper and Youtube demos

### LINKED ISSUE #1785: Add link to Arxiv preprint
**Is your feature request related to a problem? Please describe.**
I can't find a link to your Arxiv preprint in the readme of this repo.

**Describe the solution you'd like**
Please mention and add a link to your Arxiv preprint https://arxiv.org/abs/2001.06846 in the readme of this repo.

### PR DESCRIPTION
**Pull request by wangkuiyi**
Fix https://github.com/sql-machine-learning/sqlflow/issues/1785

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,3 +77,11 @@ Your feedback is our motivation to move on. Please let us know your questions, c
 ## License
 
 [Apache License 2.0](https://github.com/sql-machine-learning/sqlflow/blob/develop/LICENSE)
+
+## Published
+
+- An arXiv paper at https://arxiv.org/abs/2001.06846
```
Comment (shendiaomo) [inline README.md:83]: Add markdown links to avoid copy-n-paste?

Comment (wangkuiyi) [reply README.md:83]: I believe users can click the URL and go to the arXiv page.

File: README.md [hunk 2]
```diff
@@ -77,3 +77,11 @@ Your feedback is our motivation to move on. Please let us know your questions, c
 ## License
 
 [Apache License 2.0](https://github.com/sql-machine-learning/sqlflow/blob/develop/LICENSE)
+
+## Published
```
Comment (tonyyang-svail) [inline README.md:81]: Published or Publication?

**Review by tonyyang-svail — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (wangkuiyi) [reply README.md:81]: Not sure if Youtube videos are publications, so I used the word published.

**Comment by impredicative** [comment]
arXiv is considered a preprint, not a publication.

**Pull request merged**